desktop/apps/categories3/utils.js

```diff
@@ -8,3 +9,20 @@ export const featuredGenesForFamily = (familyName, featuredGenesList) => {
     featuredGenesFamily => featuredGenesFamily.name === familyName
   )
 }
+
+export const frameAnimator = (options, callback) => {
+  const { duration = 500, startValue = 0, endValue = 1 } = options
+  const startedAt = window.performance.now()
+
+  const animator = timestamp => {
+    const elapsed = timestamp - startedAt
+    if (elapsed <= duration) {
+      const t = elapsed / duration
+      const currentEasing = cubicInOut(t)
```

Comment:
@anandaroop - Would be cool if you could pass in an ease function or a string representation; e.g., 

```js
const scroller = frameAnimator({
  ease: 'expoInOut' 
})

// or 
import { cubicInOut } from './easing'

const scroller = frameAnimator({
  ease: cubicInOut
})
```

Reply:
Good idea, I'll add that.